service_config.py

```diff
@@ -1,13 +1,24 @@
 import os
-import re
 
 """
 The Config contains everything needed to run the service. Most entries have a default
 value and an environment value to override it.
 
 """
-allowed_domains = os.environ.get('ALLOWED_DOMAINS', 'admin.ch,swisstopo.ch,bgdi.ch').split(',')
-allowed_hosts = os.environ.get('ALLOWED_HOSTS', 'api.geo.admin.ch,api3.geo.admin.ch').split(',')
-aws_table_name = os.environ.get('AWS_DYNAMODB_TABLE_NAME', 'shortlinks_test')
-aws_region = os.environ.get('AWS_DYNAMODB_TABLE_REGION', 'eu-central-1')
-allowed_domains_pattern = re.compile(r"^.*\.admin\.ch|.*\.swisstopo\.cloud|.*\.bgdi\.ch$")
+ENV_FILE = os.getenv('ENV_FILE', None)
+if ENV_FILE:
+    from dotenv import load_dotenv
+
+    print(f"Running locally hence injecting env vars from {ENV_FILE}")
+    load_dotenv(ENV_FILE, override=True, verbose=True)
+
+ALLOWED_DOMAINS = [
+    r'.*\.geo\.admin\.ch',
+    r'.*bgdi\.ch',
+    r'.*\.swisstopo\.cloud',
+]
```

Comment:
Those default value should not be part of the repo but  read from an environment variable and part of infra. By default I would allow everything `.*`

Reply:
My bad. I think it was implemented like this before, and I edited it. But I see, we quite recently changed out template-service-flask that way. So I'll revert this changes

Reply:
Done. Now ALLOWED_DOMAINS comes from the env again. Also adapted the k8s config accordingly